lib/utils.js

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) 2021-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+
+/**
+ * Creates white spaces.
+ *
+ * @param {number} many - How many white spaces?
+ *
+ * @returns {string} A string contains white spaces.
+*/
+export const spaces = many => new Array(many).join(' ');
+
+/**
+ * Counts how many parents a test/ suite has.
```

Comment:
```suggestion
 * Counts how many parents a test suite has.
```

Reply:
this is actually correct as is. It removes from both `tests` and `suites` so `/` is an or here.

Reply:
Addressed here: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/10/commits/b31a8e80d3f62e46903c3bcc9bf561e36d4cd79d

Reply:
Cool, got confused cz of the space after / 
Thanks for clarifying.